graph/schema.resolvers.go

```diff
@@ -0,0 +1,90 @@
+package graph
+
+// This file will be automatically regenerated based on the schema, any resolver implementations
+// will be copied through when generating and any unknown code will be moved to the end.
+// Code generated by github.com/99designs/gqlgen version v0.17.43
+
+import (
+	"context"
+	"strconv"
+
+	"github.com/TanyaEIEI/pokedex/database"
+	"github.com/TanyaEIEI/pokedex/graph/model"
+)
+
+// CreatePokemon is the resolver for the createPokemon field.
+func (r *mutationResolver) CreatePokemon(ctx context.Context, input model.PokemonInput) (*model.Pokemon, error) {
+	newpokemon, err := r.Pokedex.CreatePokemon(ctx, database.Pokemon{
+		Name:        *input.Name,
+		Description: *input.Description,
+		Category:    *input.Category,
+		Abilities:   *input.Abilities,
+		Type:        *input.Type,
```

Comment:
what happen if some field of input is nil

Reply:
i fixed it